backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java

```diff
@@ -23,4 +23,11 @@ public class DeviceSupportedDisease {
   @Id
   @Column(name = "supported_disease_id", nullable = false)
   private UUID supportedDiseaseId;
+
+  @Column private String testPerformedCode;
```

Comment:
This `testPerformedCode` will also need a setter at some point - up to you if you'd like to add that now or in a future PR.

Reply:
adding `@Setter `won't be hard to add when we need it in the future, so I'll leave it like this for now.